qualtran/_infra/bloq_example.py

```diff
@@ -36,18 +38,18 @@ class BloqExample:
         generalizer: Passed to `get_bloq_counts_graph` calls for bloq-counts equivalence checking.
     """
 
-    _func: Callable[[], Bloq] = field(repr=False, hash=False)
+    _func: Callable[[], BloqType] = field(repr=False, hash=False)
     name: str
     bloq_cls: Type[Bloq]
-    generalizer: Callable[[Bloq], Optional[Bloq]] = field(
+    generalizer: Callable[[BloqType], Optional[BloqType]] = field(
```

Comment:
loving the generics, but this one actually shouldn't be `BloqType`. This function can be anything that takes bloqs to bloqs. You can use `qualtran.resource_counting.GeneralizerT` actually. And I guess this should be `Union[GeneralizerT, Sequence[GeneralizerT]]`

Reply:
Done.  Let me know if I still have it incorrect.